app/models/ont/well_factory.rb

```diff
@@ -8,7 +8,7 @@ module Ont
   class WellFactory
     include ActiveModel::Model
 
-    validate :check_well, :check_request_factories, :check_tag_service, :check_for_raised_exceptions
+    validate :check_well, :check_request_factories, :check_for_raised_exceptions
 
     def initialize(attributes = {})
       @request_factories = []
```

Comment:
we can change this so that @request_factories becomes an attr_reader i.e. 
```
def request_factories
@request_factories ||= []
end
```

Reply:
Resolved my making the well factory only have at most a single request factory